.github/workflows/ci.yaml

```diff
@@ -0,0 +1,57 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
```

Comment:
Similarly, I'm not familiar with industrial_ci. Is there a way to check that it's working before I merge? Or do I have to merge for Github to automatically run it?

Reply:
To enable Github Actions: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository

Reply:
Also please use squash merge when merging PRs, it would squash all the commits into a single commit thus we don't actually need to care about commits. History will be pretty clean this way. You can find this option on the little down button near the merge pull request button.

Reply:
OK, I'll check CI status here after merging; actions should be enabled.

And I'll squash the commits for this one =)